turtleFSI/problems/__init__.py

```diff
@@ -76,12 +77,15 @@ def info_red(s, check=True):
     mu_s=5.0E4,      # Shear modulus or 2nd Lame Coef. for the solid
     nu_s=0.45,       # Poisson ratio
     lambda_s=4.5E5,  # 1st Lame Coef. for the solid
+    k_s = 1.0E8,     # elastic response necesary for RobinBC
```

Comment:
It seems a bit weird to have non-zero default values for these constants, but then have Robin BC turned off by default. Maybe these should default to 0.0?